docs/runtimes/multiple-funcs-exithandler.ipynb

```diff
@@ -17,7 +17,15 @@
    "execution_count": 1,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        return "Function B has been triggered"
</pre><p>also remove the return after the raise - it doesn't make sense since the function will stop at the raise</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/7765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>